### PR TITLE
Initialize forking learn plugin

### PR DESCRIPTION
**Pull request by mingwli**
fork code inside docfx solution

https://ceapex.visualstudio.com/Engineering/_git/OpenPublishing.CommonPlugins?path=%2FTripleCrownValidation

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/6254)

### CODE REVIEW

File: docfx.sln [hunk 1]
```diff
@@ -13,13 +13,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "docfx.Test", "test\docfx.Te
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Docs.Template", "src\Microsoft.Docs.Template\Microsoft.Docs.Template.csproj", "{CECDC553-0AB4-4C3D-B093-F9EAD61A2948}"
 EndProject
```
Comment (yufeih) [inline docfx.sln:15]: revert this file?

File: src/TripleCrownValidation/App.config [hunk 2]
```diff
@@ -0,0 +1,98 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (yufeih) [inline src/TripleCrownValidation/App.config:1]: remove?

File: src/TripleCrownValidation/CommandLineOptions.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
+﻿using Microsoft.OpenPublishing.PluginHelper;
```
Comment (yufeih) [inline src/TripleCrownValidation/CommandLineOptions.cs:1]: Rename the folder and csproj to start with `Microsoft.Docs.LearnValidation`?

File: src/Microsoft.Docs.LearnValidation/DependencyMap/DependencyItem.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TripleCrownValidation.DependencyMap
+{
+    public class DependencyItem
```
Comment (live1206) [inline src/Microsoft.Docs.LearnValidation/DependencyMap/DependencyItem.cs:10]: [JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]

or we do not want to touch the code? Just import them

**Pull request merged**